src/lib/models/emulation/Core.cc

```diff
@@ -159,7 +159,7 @@ void Core::tick() {
   }
 
   execute(uop);
-  isa_.updateSystemTimerRegisters(&registerFileSet_, ticks_);
+  isa_.updateSystemTimerRegisters(&registerFileSet_, ticks_, ticks_);
```

Comment:
Although this is subject to change based on Finn's prior comments, the 3rd param for the number of retired instructions should be `instructionsExecuted_` not `ticks_` I believe. It's not always true that `ticks_` == `instructionsExecuted_`

Reply:
Have removed this parameter in light of the coming update to this function and the RISC-V CSR implementation. I'll keep this and the following related comments in mind when implementing this